custom-metrics-stackdriver-adapter/deploy/production/adapter.yaml

```diff
@@ -119,3 +120,45 @@ spec:
     name: custom-metrics-stackdriver-adapter
     namespace: custom-metrics
   version: v1beta1
+---
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1beta1.external.metrics.k8s.io
+spec:
+  insecureSkipTLSVerify: true
```

Comment:
The use of insecureSkipTLSVerify is strongly discouraged by the documentation. Can we not ignore TLS certificates?

Reply:
I've checked that it doesn't work when this option isn't enabled. WDYT about creating an issue to revisit the whole deployment later? I prefer not to block on this, but revisiting this will be useful in the long run, e.g. measure the resource requirements more precisely.

Reply:
sgtm, please do

Reply:
Maybe it would be better to have a separate issue for setting up TLS and for other things (like updating requests, etc)? TLS seems to be a much higher priority item than other ones.

Reply:
Filed #142 and #143